server/config.go

```diff
@@ -19,6 +19,15 @@ type MatterbuildConfig struct {
 	AllowedUsers  []string
 	ReleaseUsers  []string
 
+	SSHKeyPath string
```

Comment:
Should the new variables be prefixed by `PluginSigning` that it's clear they are only used for that? Not sure if the AWS credentials are only for Plugins as well.

Reply:
for now, yes, but we can use for other things
but will prefix that